### PR TITLE
Add version number to config dialog

### PR DESCRIPTION
**Pull request by pierce-smith1**
#25 

We're gonna have to remember to update the version string in `yokscr.h` every time we make a release. I don't know how the fuck that's going to work. There's probably some super fancy automatic github shit we could set up but it probably isn't worth it.

### CODE REVIEW

File: resourcew.h [hunk 1]
```diff
@@ -47,7 +48,7 @@
 #ifndef APSTUDIO_READONLY_SYMBOLS
 #define _APS_NEXT_RESOURCE_VALUE        159
 #define _APS_NEXT_COMMAND_VALUE         40001
-#define _APS_NEXT_CONTROL_VALUE         1020
+#define _APS_NEXT_CONTROL_VALUE         1021
```
Comment (pierce-smith1) [inline resourcew.h:51]: The Win32 API continues to entertain me with its naivety.

_SURELY there's nothing that can go wrong with making every new control have an ID that's just one larger than the previous, right? That won't possibly mean that two people working in different branches will end up with controls with the same ID and have a nightmare on their hands, right?_

Different times.

Comment (McAwesome123) [reply resourcew.h:51]: Speaking of this, I feel the need to mention that there's are `IDC_CHECK1` and `IDC_COMBO1` defines that have the same value as something else and are completely unused as far as I can tell.

**Review by McAwesome123 — APPROVED** [review]
Seems good, just don't forget to change the version number for the next release :)

File: resourcew.h (hunk 1 above)
Comment (pierce-smith1) [reply resourcew.h:51]: We should probably get rid of those, but they're most likely just artifacts from visual studio and aren't causing any trouble.

**Pull request merged**